.github/fabricbot.json

```diff
@@ -173,6 +173,25 @@
             "vsadov"
           ]
         },
+        {
+          "labels": [
+            "area-tools-linker"
```

Comment:
So is it going to be `ILLink` or `linker` ?

Reply:
https://github.com/dotnet/runtime/pull/77569#discussion_r1014525023

Reply:
I'm fine with ILLink. I'll change the label name